tensorrt_llm/_torch/pyexecutor/sampler/greedy_sample_kernels.py

```
@@ -0,0 +1,215 @@
+That is obtained without a second comparison pass by reducing a single packed
+key per candidate -- the value in the high half, the complemented index in the
+low half -- so one integer maximum resolves both.
+"""
+
+from __future__ import annotations
+
+import torch
+import triton
+import triton.language as tl
+
+__all__ = ["greedy_argmax_scatter", "supports_greedy_argmax_scatter", "ARGMAX_SPLITS"]
```

Ruff (0.16.3)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)